xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -1188,11 +1188,15 @@ IHardwareDecoder* CDVDVideoCodecFFmpeg::CreateVideoDecoderHW(AVPixelFormat pixfm
 #define VP_VIDEOCODEC_HW
 IHardwareDecoder* CDVDVideoCodecFFmpeg::CreateVideoDecoderHW(AVPixelFormat pixfmt, CProcessInfo &processInfo)
 {
+#if defined(HAVE_LIBVA)
```

Comment:
Ah I see. Yep it would be better than maintaining it in the code, but I don't think that it's in the scope of this PR since it would need quite some shuffling around also with respect to the other implementations. Will add a comment for now.